src/ActionList.jsx

```diff
@@ -19,6 +20,31 @@ export default class ActionList extends Component {
 
   componentDidMount() {
     this.scrollToBottom(true);
+    if (!this.props.draggableActions) return;
+    const container = ReactDOM.findDOMNode(this.refs.rows);
+    this.drake = dragula([container], {
+      copy: false,
+      copySortSource: false,
+      mirrorContainer: container,
+      accepts: (el, target, source, sibling) => (
+        !sibling || parseInt(sibling.getAttribute('data-id'))
+      ),
+      moves: (el, source, handle) => (
+        parseInt(el.getAttribute('data-id')) &&
+        handle.className.indexOf('selectorButton') !== 0
```

Comment:
We're returning `false` here, not to drag through the buttons. It could be `=== -1`, but it doesn't guarantee that it's a substring of another class. On the other hand, it implies that this class will be always the first.